kafe2/fit/histogram/fit.py

```diff
@@ -7,9 +7,9 @@
 from ...config import kc
 from ...core import NexusFitter, Nexus
 from ...core.fitters.nexus import Parameter, Alias, NexusError
-from .._base import FitException, FitBase, DataContainerBase, CostFunctionBase
+from .._base import FitException, FitBase, DataContainerBase
 from .container import HistContainer
-from .cost import HistCostFunction_NegLogLikelihood, HistCostFunction_UserDefined, STRING_TO_COST_FUNCTION
+from .._base.cost import CostFunction_NegLogLikelihood, CostFunction, STRING_TO_COST_FUNCTION
```

Comment:
Keeping the cost functions for more intuitive use. As discussed previously.

Reply:
Sorry, this is actually something that I forgot to change back.
The fit is supposed to use HistCostFunction.

Reply:
Disregard my previous comment.
I forgot that I decided to use the base cost function classes in the fit classes so that we need to maintain only one instance of _STRING_TO_COST_FUNCTION.
If we ever need to implement functionality specific to hist or indexed it should be changed.